pkg/sql/show.go

```diff
@@ -659,6 +659,77 @@ func (p *planner) ShowTables(ctx context.Context, n *parser.ShowTables) (planNod
 		initialCheck, nil)
 }
 
+// ShowSyntax implements the plan for SHOW SYNTAX. This statement is
+// usually handled as a special case in Executor, but for
+// FROM [SHOW SYNTAX ...] we will arrive here too.
+func (p *planner) ShowSyntax(ctx context.Context, n *parser.ShowSyntax) (planNode, error) {
+	var query bytes.Buffer
+	query.WriteString("SELECT @1 AS field, @2 AS text FROM (VALUES ")
+
+	first := true
+	if err := runShowSyntax(ctx, n.Statement, func(ctx context.Context, field, msg string) error {
+		if !first {
+			query.WriteString(", ")
+		}
+		first = false
+		fmt.Fprintf(&query, "('%s', %s)", field, parser.EscapeSQLString(msg))
```

Comment:
fmt.Fprintf(&query, "%s('%s', %s)", comma, field, parser.EscapeSQLString(msg))

where comma := "" and then is made ", " after the first iteration

Reply:
hmmm, Actually probably better to add the strings into a []string and then call join after method